src/coreComponents/python/modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py

```diff
@@ -0,0 +1,136 @@
+
+import meshio
+from meshio._mesh import CellBlock
+import numpy as np
+import argparse
+import logging
+
+
+def convert_abaqus_to_gmsh(input_mesh, output_mesh, logger=None):
+    """
+    @brief Convert an abaqus mesh to gmsh 2 format, preserving nodeset information.
+    @details If the code encounters any issues with region/element indices,
+             the conversion will attempt to continue, with errors
+             indicated by -1 values in the output file.
+    @param input_mesh path of the input abaqus file
+    @param output_mesh path of the output gmsh file
+    @param logger an instance of logging.Logger
+    """
+    # Initialize the logger if it is empty
+    if not logger:
+      logging.basicConfig(level=logging.WARNING)
+      logger = logging.getLogger(__name__)
+
+    # Load the mesh
+    logger.info('Reading abaqus mesh...')
+    mesh = meshio.Mesh.read(input_mesh, "abaqus")
```

Comment:
Sorry again, but it's more common to use the function ``meshio.read``:

```python
mesh = meshio.read(input_mesh, file_format="abaqus")
```

or

```python
mesh = meshio.abaqus.read(input_mesh)
```

Reply:
I agree that looks a bit cleaner